src/app.service.ts

```diff
@@ -1,8 +0,0 @@
-import { Injectable } from '@nestjs/common';
-
-@Injectable()
-export class AppService {
-  getHello(): string {
-    return 'Hello World!';
-  }
```

Comment:
Usually it is good to leave a simple endpoint for testing the api server is online, the monitoring tools call it to check if server is online